common/src/main/java/org/opensearch/ml/common/settings/MLCommonsSettings.java

```diff
@@ -342,4 +342,12 @@ private MLCommonsSettings() {}
     /** This setting sets the remote metadata service name */
     public static final Setting<String> REMOTE_METADATA_SERVICE_NAME = Setting
         .simpleString("plugins.ml_commons." + REMOTE_METADATA_SERVICE_NAME_KEY, Setting.Property.NodeScope, Setting.Property.Final);
+
+    // Feature flag for enabling telemetry metric collection via metrics framework
```

Comment:
what's the difference between static metric vs metric?

Reply:
From the name `metrics_collection_enabled` should include `metrics_static_collection_enabled`. If user enable `metrics_collection_enabled`, the `metrics_static_collection_enabled` also enabled?